### PR TITLE
Fix logical error after demo review

### PR DESCRIPTION
**Pull request by Activict**

### CODE REVIEW

File: StoreApp.BLL/Services/BrandService.cs [hunk 1]
```diff
@@ -70,6 +70,16 @@ public int GetCountProductsByBrandId(int id)
             return DataBase.Products.GetAll().Count(p => p.BrandId.Equals(id));
         }
 
+        public bool CheckExistBrand(BrandDTO brandDTO)
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/BrandService.cs]: First of all I propose to name methods which returns boolean like **Is...**
Some examples 
https://stackoverflow.com/questions/1370840/naming-conventions-what-to-name-a-method-that-returns-a-boolean

File: StoreApp.BLL/Services/CategoryService.cs [hunk 2]
```diff
@@ -74,6 +74,16 @@ public int GetCountProductsByCategoryId(int id)
             return DataBase.Products.GetAll().Count(p => p.CategoryId.Equals(id));
         }
 
+        public bool CheckExistCategory(CategoryDTO categoryDTO)
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/CategoryService.cs]: The same

File: StoreApp.BLL/Services/ProducerService.cs [hunk 3]
```diff
@@ -72,6 +72,16 @@ public int GetCountProductsByProducerId(int id)
             return DataBase.Products.GetAll().Count(p => p.ProducerId.Equals(id));
         }
 
+        public bool CheckExistProducer(ProducerDTO producerDTO)
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/ProducerService.cs]: The same

File: StoreApp.BLL/Services/UnitService.cs [hunk 4]
```diff
@@ -76,6 +76,16 @@ public int GetCountProductsByUnitId(int id)
             return DataBase.Products.GetAll().Count(p => p.UnitId.Equals(id));
         }
 
+        public bool CheckExistUnit(UnitDTO unitDTO)
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/UnitService.cs]: The same

File: StoreApp/Controllers/AccountController.cs [hunk 5]
```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
...
-                                                                  : UserRoles.admin.ToString();
+                var i = UserRoles.user.ToString();
+                user.Role = int.Parse(user.Role) == (int)(UserRoles.user) ? UserRoles.user.ToString() 
+                                                                          : UserRoles.admin.ToString();
 
                 UserDTO userDTO = webMapper.config.Map<UserViewModel, UserDTO>(user);
 
                 userService.Edit(userDTO);
 
+                TempData["StatusMessage"] = "success";
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/AccountController.cs]: It's better to add some Enum to keep the values (success, danger).
This tip can be applied to the next statements below.

File: StoreApp/Views/Store/Index.cshtml [hunk 6]
```diff
@@ -101,16 +104,27 @@
...
+                else
+                {
+                    <div class="col-2">@Html.ActionLink("Buy", "Login", "Account", new { id = item.Id }, new { @class = "btn btn-success " })</div>
+                }
+
+                @if (role != null && role.Equals("admin"))
+                {
+                    <div class="col-2">
+                        @Html.ActionLink("Edit", "EditProduct", new { id = item.Id })
+                        @Html.ActionLink("Del", "DeleteProduct", new { id = item.Id })
```
Comment (AleksandroBoko) [inline StoreApp/Views/Store/Index.cshtml]: It is better to write "**Delete**"

File: StoreApp/Views/Store/Index.cshtml [hunk 7]
```diff
@@ -5,18 +5,21 @@
...
-<br />
+
+@if (TempData["Message"] != null)
+{
+    <span class="offset-2 badge badge-@TempData["StatusMessage"]">@TempData["Message"]</span>
+}
+else
+{
+    <br />
+}
```
Comment (AleksandroBoko) [inline StoreApp/Views/Store/Index.cshtml:23]: Are you sure you need to keep this block? 
Btw you can move tag **br** to the next line after if

File: StoreApp/Controllers/AccountController.cs [hunk 8]
```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
         {
             if ((Session["Role"] as string).Equals(UserRoles.admin.ToString()))
             {
-                user.Role = user.Role.Equals((int)UserRoles.user) ? UserRoles.user.ToString() 
-                                                                  : UserRoles.admin.ToString();
+                var i = UserRoles.user.ToString();
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/AccountController.cs]: Why do you need to set this variable? What's the reason?

File: StoreApp/Controllers/AccountController.cs [hunk 9]
```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
         {
             if ((Session["Role"] as string).Equals(UserRoles.admin.ToString()))
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/AccountController.cs]: I propose to change 
**admin** to **Admin**
**user** to **User**
It looks better

File: StoreApp/Controllers/AccountController.cs [hunk 10]
```diff
@@ -156,7 +157,7 @@ public ActionResult EditUser(UserEditViewModel userEdit)
             if (userValidateService.CheckForEditUser(userDTO))
             {
                 userService.Edit(userDTO);
-
+                TempData["StatusMessage"] = "success";
                 TempData["Message"] = "Your edit account successful!";
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/AccountController.cs]: It's better to write
"The user was updated successfully"

File: StoreApp/Controllers/AccountController.cs [hunk 11]
```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
...
+                var i = UserRoles.user.ToString();
+                user.Role = int.Parse(user.Role) == (int)(UserRoles.user) ? UserRoles.user.ToString() 
+                                                                          : UserRoles.admin.ToString();
 
                 UserDTO userDTO = webMapper.config.Map<UserViewModel, UserDTO>(user);
 
                 userService.Edit(userDTO);
 
+                TempData["StatusMessage"] = "success";
                 TempData["Message"] = $"{user.UserName} edit success";
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/AccountController.cs]: message: **The UserName was updated successfully**
Check the messages below please

File: StoreApp/Views/Shared/_Layout.cshtml [hunk 12]
```diff
@@ -43,7 +43,10 @@
             <ul class="navbar-nav">
                 @if (Request.IsAuthenticated)
                 {
-                    <li class="nav-item">@Html.Action("CartPartial", "Cart")</li>
+                    if (!Session["Role"].ToString().Equals("admin"))
```
Comment (AleksandroBoko) [inline StoreApp/Views/Shared/_Layout.cshtml]: You can call Enum here. 
Check this statement below please

File: StoreApp/Controllers/AccountController.cs (hunk 11 above)
Comment (Activict) [reply StoreApp/Controllers/AccountController.cs]: fixed

File: StoreApp/Controllers/AccountController.cs (hunk 10 above)
Comment (Activict) [reply StoreApp/Controllers/AccountController.cs]: fixed

File: StoreApp.BLL/Services/BrandService.cs (hunk 1 above)
Comment (Activict) [reply StoreApp.BLL/Services/BrandService.cs]: fixed

File: StoreApp.BLL/Services/CategoryService.cs (hunk 2 above)
Comment (Activict) [reply StoreApp.BLL/Services/CategoryService.cs]: fixed

File: StoreApp.BLL/Services/ProducerService.cs (hunk 3 above)
Comment (Activict) [reply StoreApp.BLL/Services/ProducerService.cs]: fixed

File: StoreApp.BLL/Services/UnitService.cs (hunk 4 above)
Comment (Activict) [reply StoreApp.BLL/Services/UnitService.cs]: fixed

File: StoreApp/Controllers/AccountController.cs (hunk 5 above)
Comment (Activict) [reply StoreApp/Controllers/AccountController.cs]: fixed

File: StoreApp/Controllers/AccountController.cs (hunk 8 above)
Comment (Activict) [reply StoreApp/Controllers/AccountController.cs]: fixed

File: StoreApp/Controllers/AccountController.cs (hunk 9 above)
Comment (Activict) [reply StoreApp/Controllers/AccountController.cs]: fixed

File: StoreApp/Views/Shared/_Layout.cshtml (hunk 12 above)
Comment (Activict) [reply StoreApp/Views/Shared/_Layout.cshtml]: fixed

File: StoreApp/Views/Store/Index.cshtml (hunk 6 above)
Comment (Activict) [reply StoreApp/Views/Store/Index.cshtml]: fixed

File: StoreApp/Views/Store/Index.cshtml (hunk 7 above)
Comment (Activict) [reply StoreApp/Views/Store/Index.cshtml:23]: this stylistic decision

**Pull request merged**